composer/datasets/streaming/world.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2022 MosaicML. All Rights Reserved.
+
+"""The :class:`World` class is used for easily querying distributed training info, used by `StreamingDataset`.
+"""
+
+from typing import NamedTuple
+
+from torch.utils.data import get_worker_info
+
+from composer.utils import dist
+
+__all__ = ["World", "get_world"]
+
+
+class World(NamedTuple):
+    """A :class:`NamedTuple` that provides context about workers, devices, and nodes."""
```

Comment:
Can you add a docstring line under the "Attributes:" section for each field in the namedtuple? See other namedtuples in Composer as examples.